redwood-cli/src/main/kotlin/app/cash/redwood/generator/sharedHelpers.kt

```diff
@@ -111,6 +111,18 @@ internal fun Schema.layoutModifierImpl(layoutModifier: LayoutModifier): ClassNam
 internal val Schema.toLayoutModifier: MemberName get() =
   MemberName(widgetPackage(this), "toLayoutModifier")
 
+internal val Schema.toJsonArray: MemberName get() =
+  MemberName(composePackage(this), "toJsonArray")
+
+internal val Schema.toJsonElement: MemberName get() =
+  MemberName(composePackage(this), "toJsonElement")
+
+internal fun Schema.allLayoutModifiers(): List<Pair<Schema, LayoutModifier>> {
```

Comment:
Was thinking about making this a map, but decided against it since I wanted an ordered collection.

Reply:
Someday https://openjdk.org/jeps/431#Sequenced-maps

Reply:
If it's all internal code you can use the implementation type `LinkedHashMap` to guarantee iteration order is preserved / exposed.

Reply:
Ooh yeah sequenced collections would be perfect.

Reply:
Opted not to use a `Map` in the end since we need to know the total number of all layout modifiers.